doc.go

```diff
@@ -382,6 +388,25 @@ of pigeon and should not be used nor modified. Those keys are treated as
 internal implementation details and therefore there are no guarantees given in
 regards of API stability.
 
+Left recursion
+
+With options -support-left-recursion pigeon supports left recursion. E.g.:
+	expr = expr '*' term
+Supports indirect recursion:
+	A = B / D
+	B = A / C
+Preserve support for ordered choices:
```

Comment:
Sorry my fault, an `s` is missing:

```suggestion
Preserves support for ordered choices:
```

Reply:
Not applicable